src/main/java/com/github/markusbernhardt/seleniumlibrary/keywords/Element.java

```diff
@@ -934,8 +934,8 @@ public void xpathShouldMatchXTimes(String xpath, int expectedXpathCount, String.
     @RobotKeyword("Click to element from list elements by locator ``xpath``.")
     @ArgumentNames({"xpath", "index=0", "message=NONE"})
     public void clickElementByIndex(String xpath, String... params) {
-        String message = robot.getParamsValue(params, 0, "");
-        int index = robot.getParamsValue(params, 1, 0);
+        int index = robot.getParamsValue(params, 0, 0);
+        String message = robot.getParamsValue(params, 1, "");
```

Comment:
Did you fix a bug here? Or... what has been changed here then?

Reply:
Yep, fixed